source/ContentScript/recorder.ts

```diff
@@ -26,6 +26,7 @@ import {
   ZestStatementElementSubmit,
   ZestStatementLaunchBrowser,
   ZestStatementSwitchToFrame,
+  ZestStatementElementScrollTo,
```

Comment:
Better keep in alphabetic order, same in the other places.

Reply:
Oh sorry I didn't notice